### DISCUSSION TITLE
grid arbitrary value not working

### DISCUSSION THREAD

**Discussion by Beenhakker** [tailwindlabs/tailwindcss#11001] [Help]
Hi, 

Similar questions have asked before, but I cannot seem to get this to work:

`<div class="grid grid-cols-[repeat(auto-fit, minmax(300px, 1fr))] grid-flow-row auto-rows-[minmax(300px, auto)]"></div>`

More simple values like `auto_auto` are correctly accepted.

**Comment by adamwathan** [comment]
Hey! Classes can't have spaces, need to use an underscore or remove the space:

```
<div class="grid grid-cols-[repeat(auto-fit,minmax(300px,1fr))] grid-flow-row auto-rows-[minmax(300px,auto)]"></div>
```

With your input, the browser sees these classes:

`grid-cols-[repeat(auto-fit,`
`minmax(300px, 1fr))]`
`auto-rows-[minmax(300px,`
`auto)]`
